gc/stats/CollectionStatistics.hpp

```diff
@@ -45,6 +45,12 @@ class MM_CollectionStatistics : public MM_Base
 	uint64_t  _endTime;			/**< Collection end time */
 	uint64_t  _stallTime;		/**< Collection stall time */
 
+	uint64_t  _pauseStart;		/**< When STW starts */
+	uint64_t  _pauseEnd;		/**< When STW ends */
```

Comment:
actually, we don't really need to record these. we can just directly deal with the original data
hence, no need for recordPauseStart(), either